content/well-architected-framework/docs/docs/secure-systems/infrastructure/secure-access.mdx

```diff
@@ -0,0 +1,209 @@
+---
+
+# Secure human access to infrastructure
+
+The move to cloud-native and ephemeral resources has changed how
+people think about securing access to infrastructure. Regardless of your
+deployment methodology, your teams still need to access infrastructure for
+troubleshooting and incident response.
+
+In traditional infrastructure deployments, you might have used bastion hosts or VPNs
+to provide secure access to infrastructure. However, these methods can introduce
+additional attack surfaces and may not align with modern security best
+practices. Managing access to infrastructure typically relied on long-lived,
+static credentials. Long-lived credentials increase the risk of compromise,
+leading to unauthorized access.
+
+## Why secure human access to infrastructure?
+
+Securing human access to infrastructure is critical to protect against
+unauthorized access and potential threats. Organizations must implement a strong
+identity and access management program, including:
+
+- [**Define access
+  requirements:**](/well-architected-framework/secure-systems/identity-access-management/define-access-requirements)
+  to ensure only authorized users can access infrastructure.
+- [**Ensure access follows the principle of least
+  privilege:**](/well-architected-framework/secure-systems/identity-access-management/grant-least-privilege)
+  to ensure users have only the minimum access necessary to perform their job functions.
+- [**Centralize authentication and
+  authorization:**](/well-architected-framework/secure-systems/identity-access-management/centralize-identity-management)
+  to protect against unauthorized access and simplify identity management.
+- [**Use strong authentication
+  methods:**](/well-architected-framework/secure-systems/identity-access-management/implement-strong-sign-in-workflows)
+  to protect against credential theft and unauthorized access.
+- [**Switch to ephemeral, dynamic
+  credentials:**](/well-architected-framework/secure-systems/identity-access-management/use-dynamic-credentials)
+  to reduce the risk of credential compromise. 
+- [**Audit and manage account
+  access:**](/well-architected-framework/secure-systems/identity-access-management/manage-access-lifecycle)
+  to ensure access is reviewed and revoked when no longer needed.
+
+Implementing an effective identity and access management program sets the
+foundation to secure human access to infrastructure.
+
+## Deploy access controls with Terraform
+
+HashiCorp Terraform helps you adapt to how you manage and audit infrastructure
+access. Instead of platform engineers accessing infrastructure directly, they
+use Terraform to define and manage infrastructure as code. 
+
+- **Eliminates direct console access** - Teams interact with infrastructure through code review and approval processes rather than direct cloud console access
```

Comment:
```suggestion
- **Eliminates direct console access:** Teams interact with infrastructure through code review and approval processes rather than direct cloud console access
```